frontend/server/src/DAO/QualityNominations.php

```diff
@@ -480,6 +480,33 @@ public static function calculateGlobalDifficultyAndQuality(array $contents): arr
         return [$qualitySum / $qualityN, $difficultySum / $difficultyN];
     }
 
+    /**
+     * This function gets contents of QualityNomination table
+     *
+     * @return \OmegaUp\DAO\VO\QualityNominations[]
```

Comment:
nit:

```suggestion
     * @return list<\OmegaUp\DAO\VO\QualityNominations>
```

en general, `list<T>` se prefiere sobre `T[]`.

Reply:
listo